analyzer/javacg-wala/pom.xml

```diff
@@ -67,22 +67,22 @@
         </dependency>
 
         <!-- Uncomment the slf4j package only if you wanna use WALA module outside the FASTEN server -->
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-simple</artifactId>
-            <version>1.7.30</version>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>1.7.30</version>
-        </dependency>
-        <dependency>
-            <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-engine</artifactId>
-            <version>5.4.0</version>
-            <scope>test</scope>
-        </dependency>
+<!--        <dependency>-->
```

Comment:
Please remove dead code

Reply:
This is not dead code. It is commented out because it interferes with the server dependencies but if you want to run it as a standalone plugin, these dependencies are needed. So we just have to uncomment them whenever we want to run Main class of the plugin and they are commented out the rest of time.